src/screens/components/Contact.js

```diff
@@ -1,37 +1,41 @@
 import React from 'react'
 import styled from 'styled-components'
 
+import { accent2, secondary1 } from '@/style-guide/colors'
+
 const StyledContainer = styled.div`
-  width: 80%;
+  height: 100px;
+  width: 100%;
+  background-color: ${accent2};
+  display: flex;
+  justify-content: center;
+  align-items: center;
   font-family: "Hammersmith One";
-  color: #bb7a8c;
-  text-transform: uppercase;
-  text-align: center;
-  font-size: 1.4em;
-  margin: 60px 0;
 
-  a {
-    color: black;
+  p {
+    font-weight: bold;
+    color: ${secondary1};
   }
-`
 
-const StyledParagraph = styled.div`
-  margin: 16px 0;
+  a {
+    font-weight: bold;
+    color: ${secondary1};
+    text-decoration: none;
 
-  span {
-    display: block;
+    a:hover {
+      cursor: pointer;
+      opacity: 0.8;
+    }
   }
 `
 
 const Contact = () => (
   <StyledContainer>
-    <StyledParagraph>
-      <span>
-        CUSTOM DESTINATIONS AVAILABLE UPON REQUEST.
-        CONTACT US {' '}<a href='mailto:concierge@luxette.co'>HERE</a>{' '}
-        TO GET STARTED.
-      </span>
-    </StyledParagraph>
+    <p>
+      CUSTOM DESTINATIONS ARE AVAILABLE UPON REQUEST.
+      CONTACT US {' <'}<a href='mailto:concierge@luxette.co'>HERE</a>{'> '}
```

Comment:
Oh!!!!! that's how you do it :) lolol spent 10 mins trying to figure out the <> on both sides of the HERE on Friday :)